python/cdp-agentkit-core/cdp_agentkit_core/actions/get_balance.py

```diff
@@ -18,10 +19,14 @@ class GetBalanceInput(BaseModel):
         ...,
         description="The asset ID to get the balance for, e.g. `eth`, `0x036CbD53842c5426634e7929541eC2318f3dCF7e`",
     )
+    address_to_fund: Optional[str] = Field(
+        None,
+        description="The address to fund. If not provided, the default address of the wallet will be used.",
```

Comment:
Please see suggestion
```suggestion
        description="The address to get the balance for. If not provided, addresses of the wallet will be used.",
```